src/user/index.js

```diff
@@ -211,6 +211,10 @@ User.getAdminsandGlobalModsandModerators = async function () {
 	return await User.getUsersData(_.union(...results));
 };
 
+User.getFirstAdminUid = async function () {
+	return (await db.getSortedSetMembers('group:administrators:members')).reverse()[0];
```

Comment:
This can be changed to 
`return (await db.getSortedSetRange("group:administrators:members", 0, 0))[0];`
No need to load all members of the group.

Reply:
Fixed, thanks!